bin/zopen-build

```diff
@@ -1832,13 +1883,42 @@ install()
 
       #TODO: Hack so that we can use coreutils md5sum without impacting builds
       ZOPEN_DEPS="${ZOPEN_DEPS} coreutils jq"
+      if [ -n "${signPax}" ]; then
+              if [ -z "${ZOPEN_GPG_SECRET_KEY_FILE}" ] || [ -z "${GPG_SECRET_KEY_PASSPHRASE}" ] || [ -z "${GPG_PUBLIC_KEY_PATH}" ]; then
```

Comment:
Can you also check if GPG_PUBLIC_KEY_PATH and ZOPEN_GPG_SECRET_KEY_FILE are readable (-r)